prog/mutation.go

```diff
@@ -374,6 +374,30 @@ func (t *BufferType) mutate(r *randGen, s *state, arg Arg, ctx ArgCtx) (calls []
 	case BufferText:
 		data := append([]byte{}, a.Data()...)
 		a.data = r.mutateText(t.Text, data)
+	case BufferCompressed:
+		data := a.Data()
```

Comment:
I would also change (t *BufferType) getMutationPrio() to return maxPriority for BufferCompressed.
Such heuristic is somewhat hand-wavy, but I think it makes sense. Will also tract attention of future readers of the code.